allofplos/plos_corpus.py

```diff
@@ -433,7 +433,7 @@ def check_for_uncorrected_proofs(directory=newarticledir, text_list=uncorrected_
     for article_file in articles:
         article = Article.from_filename(article_file)
         article.directory = directory
-        if article.proof == 'uncorrected_proof':
+        if article.proof == 'uncorrected_proof' and article.doi not in uncorrected_proofs_list:
```

Comment:
This seems more complicated than necessary. Why do you need to check for either? This feels like some layer of abstraction is missing